packages/main/test/sap/ui/webcomponents/main/pages/DatePicker.html

```diff
@@ -96,12 +96,12 @@ <h3>japanese calendar type</h3>
 			console.log('change: ', e.detail);
 			labelChange.innerHTML = 'change: ' + JSON.stringify(e.detail);
 		});
-		dp.addEventListener('ui5-liveChange', function(e) {
-			console.log('liveChange: ', e.detail);
-			labelLiveChange.innerHTML = 'liveChange: ' + JSON.stringify(e.detail);
+		dp.addEventListener('input', function(e) {
```

Comment:
could you please merge the master branch? the tests are running in no-conflict mode so the events have to be with the ui5- prefix.